### ISSUE TITLE
useInfiniteScroll doesn't work with `window`

### ISSUE THREAD

**Issue by innocenzi** [vueuse/vueuse#3060]
### Describe the bug

Using `useInfiniteScroll` with `window` triggers an exception: `Uncaught TypeError: Window.getComputedStyle: Argument 1 does not implement interface Element.`

I think the issue appeared with https://github.com/vueuse/vueuse/commit/d3a2bcaeca2cce269c3bf070582da132ab2ec568. It uses `useScroll` which uses `getComputedStyle` on `window.documentElement` which is illegal, hence triggering the error.

### Reproduction

https://stackblitz.com/edit/vitejs-vite-5dmbm2?file=src/App.vue

### System Info

```Shell
System:
    OS: macOS 13.3.1
    CPU: (10) arm64 Apple M1 Pro
    Memory: 582.61 MB / 16.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 20.0.0 - /opt/homebrew/bin/node
    Yarn: 1.22.18 - /opt/homebrew/bin/yarn
    npm: 9.6.4 - /opt/homebrew/bin/npm
  Browsers:
    Chrome: 112.0.5615.137
    Firefox: 112.0.2
    Safari: 16.4
  npmPackages:
    @vueuse/components: ^10.1.2 => 10.1.2 
    @vueuse/core: ^10.1.2 => 10.1.2 
    @vueuse/head: ^1.1.26 => 1.1.26 
    @vueuse/motion: 2.0.0-beta.12 => 2.0.0-beta.12 
    vue: ^3.2.47 => 3.2.47
```

### Used Package Manager

pnpm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/vueuse/vueuse/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guidelines](https://github.com/vueuse/vueuse/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://vueuse.org/guide).
- [X] Check that there isn't [already an issue](https://github.com/vueuse/vueuse/issues) that reports the same bug to avoid creating a duplicate.
- [X] Make sure this is a VueUse issue and not a framework-specific issue. For example, if it's a Vue SFC related bug, it should likely be reported to https://github.com/vuejs/core instead.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/vueuse/vueuse/discussions).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by smallnine9** [comment]
will fix soon

**Comment by innocenzi** [comment]
Probably fixed by #3086, will re-open if next release doesn't actually fix the issue. Thanks @smallnine9 👍
